src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-notification-list/rgw-bucket-notification-list.component.html

```diff
@@ -0,0 +1,37 @@
+  <fieldset>
+    <legend i18n>
+    Notification Configuration
+    <cd-help-text>
+      Create a notification configuration to trigger alerts for specific bucket events, such as object creation or transitions, based on prefixes or tags.
+    </cd-help-text>
+  </legend>
+  </fieldset>
+  <ng-container>
+  <cd-table #table
+            [data]="notification$ | async"
+            [columns]="columns"
+            columnMode="flex"
+            selectionType="single"
+            (updateSelection)="updateSelection($event)"
+            identifier="Id"
+            (fetchData)="loadNotification($event)">
+  </cd-table>
+  </ng-container>
+  <ng-template #filterTpl
+               let-config="data.value">
+    <ng-container *ngIf="config">
+      <ng-container *ngFor="let key of ['S3Key', 'S3Metadata', 'S3Tags']">
+        <ng-container *ngIf="config[key]?.FilterRule?.length">
+          <div class="cds--label">
+            {{ key }}:
+          </div>
+          <div [cdsStack]="'horizontal'"
+               *ngFor="let rule of config[key].FilterRule">
+            <cds-tag size="sm"
+                     [type]="'blue'">{{ rule.Name }}: {{ rule.Value }}</cds-tag>
+          </div>
+          <br>
+        </ng-container>
+      </ng-container>
```

Comment:
`['S3Key', 'S3Metadata', 'S3Tags']` -> Maintaing this list in the front-end would mean that we will have to update this as the back-end team decides to support more filters which is a little annoying. We can use [keyvalue](https://angular.dev/api/common/KeyValuePipe) pipe here to parse the configuration and get the filters from it.

```suggestion
      <ng-container *ngFor="let item of config | keyvalue">
        <ng-container *ngIf="item.value?.FilterRule?.length">
          <div class="cds--label">
            {{ item.key }}:
          </div>
          <div [cdsStack]="'horizontal'"
               *ngFor="let rule of item.value.FilterRule">
            <cds-tag size="sm"
                     [type]="'blue'">{{ rule.Name }}: {{ rule.Value }}</cds-tag>
          </div>
          <br>
        </ng-container>
      </ng-container>
```